sota-client/docker/sota_provision.sh

```diff
@@ -42,6 +43,16 @@ wait_for_ntp() {
   done
 }
 
+prepare_creds() {
+  # Prepare credentials here since aktualizr no longer requires unzipping.
+  if [ -n "${in_reg}.zip" ]; then
```

Comment:
Shouldn't this be a -f, not -n? Even if $in_reg is undefined, the string will be ".zip", so this test always passes.